examples/ord-min-max.js

```diff
@@ -0,0 +1,42 @@
+const {Max, Min, Ord} = require('../fantasy-monoids');
```

Comment:
I would suggest to do it now because otherwise you are just creating additional tech debt and make the eventual switch more painful because there is yet another project that needs to be taken care of ;-)

Just my 2cents as an "outsider" ;)